### DISCUSSION TITLE
liquid-ajax-cart.js.org

### DISCUSSION THREAD

**Discussion by WalterEight** [EvgeniyMukhamedjanov/liquid-ajax-cart#36] [General]
Evgeniy,

Brillant, works over the top of other themes ok too. 
Thank you.
Also, is there a way to get the total ORIGINAL item prices ( comparitive price total ) out of the shopify API, or do we have to loop thru the
{% assign total_saving = 0 %}
{% for item in cart.items %}
  {% if item.variant.compare_at_price > item.variant.price %}
  	{% assign saving = item.variant.compare_at_price | minus: item.variant.price | times: item.quantity %}
  	{% assign total_saving = saving | plus: total_saving %}
  {% endif %}
{% endfor %}

is there a way to have this liquid refresh when a change in "+"  or "-"  is detected ? like a slide cart refresh ?

I need the cart to show "saved $XXX" under the subtotal of the slide in cart for all the items that have a compare price higher than the price.

Thanks for you help.

**Comment by EvgeniyMukhamedjanov** [comment]
Hey @WalterEight,

I think the only way to get product prices in Shopify using API is the [Product API](https://shopify.dev/api/ajax/reference/product) but if you are going to use it, you'll need to go through all the line_items, get product ids and send request for each product. Thus you'll have few Ajax requests after each cart update, that is not good I believe. Another way that you suggested using liquid loop is much more preferrable.

So if you use liquid, you don't need to detect '+' or '-' clicks because it will be calculated on liquid as well!

Liquid loop is much much easier and better in terms of performance.
